pyvista/plotting/colors.py

```diff
@@ -1204,51 +1213,25 @@ def get_cmap_safe(cmap):
             except AttributeError:
                 pass
 
-        # Else use Matplotlib
-        if not has_module('matplotlib'):
-            raise ImportError(
-                'The use of custom colormaps requires the installation of matplotlib.'
-            )  # pragma: no cover
-
-        try:
-            from matplotlib import colormaps, colors
-        except ImportError:  # pragma: no cover
-            import matplotlib
-            import scooby
-
-            min_req = '3.5.0'
-            if not scooby.meets_version(matplotlib.__version__, min_req):
-                raise ImportError(
-                    'The use of custom colormaps requires the installation of '
-                    f'matplotlib>={min_req}.'
-                ) from None
-
         if not isinstance(cmap, colors.Colormap):
-            cmap = colormaps[cmap]
+            if hasattr(colormaps, cmap):
+                # Backwards compatibility with matplotlib<3.5.0
+                cmap = getattr(colormaps, cmap)  # pragma: no cover
```

Comment:
The bug is that here we don't distinguish "`colormaps` doesn't have _any_ colormap attrs" from `colormaps` doesn't have _this_ colormap attr". We probably have to add some exception handling here. Perhaps reverse the logic and check for `TypeError` on `colormaps[cmap]` first? Not sure what the best angle is.

Reply:
Resolved in 69075a6da6b40b9bbd5731e0792d6567e2329682. Errors are now consistent; `ValueError` is always raised.